LC-1493 Deprecate/remove avatar in update/create_agent methods (#1493)
livechat/livechat-public-docs: src/pages/management/changelog/index.mdx

## [v3.6] - Developer preview

### Agents

> - The [**Create Agent**](/management/configuration-api/v3.6/#create-agent) property `avatar` is removed. Use [**Update Account Organization Settings**](/authorization/global-accounts-api#tag/AccountOrganizationSettings/paths/~1accounts~1%7Baccount_id%7D~1organization_settings~1%7Borganization_id%7D/patch) to update `avatar`.

Reviewer: ```suggestion
- The `avatar` property in [**Create Agent**](/management/configuration-api/v3.6/#create-agent) was removed. Use [**Update Account Organization Settings**](/authorization/global-accounts-api#tag/AccountOrganizationSettings/paths/~1accounts~1%7Baccount_id%7D~1organization_settings~1%7Borganization_id%7D/patch) to update `avatar`.
```

> - The [**Update Agent**](/management/configuration-api/v3.6/#update-agent) property `name` and `avatar` are now removed.

Reviewer: ```suggestion
- The `name` and `avatar`  properties in [**Update Agent**](/management/configuration-api/v3.6/#update-agent) were removed.
```
